storage/volume/types/volume_util.go

```diff
@@ -0,0 +1,229 @@
+package types
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/alibaba/pouch/pkg/bytefmt"
+	"github.com/alibaba/pouch/storage/volume/types/meta"
+
+	"github.com/pborman/uuid"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+)
+
+func translateSelector(k, v string) SelectorRequirement {
+	values := strings.Split(v, ",")
+
+	return SelectorRequirement{
+		Key:      k,
+		Operator: selection.In,
+		Values:   values,
+	}
+}
+
+func buildVolumeConfig(options map[string]string) (*VolumeConfig, error) {
```

Comment:
I think we could add some unit test for this file.

Reply:
agree it